nxt/builtin/sub_graphs.nxt

```diff
@@ -5,12 +5,93 @@
     "mute": false,
     "solo": false,
```

Comment:
```suggestion
    "solo": false,
    "meta_data": {
        "positions": {
            "/_maya_standalone_graph": [
                0.0,
                40.0
            ]
        }
    },
```

Reply:
My mistake, I misread the hierarchy before, I didn't realize the `/_maya_standalone_graph` node was at the root. This just moves it down so the two nodes don't cover each other.